third_party/expect/src/lib.rs

```diff
@@ -51,9 +53,9 @@ macro_rules! expect_matches {
 macro_rules! expect {
     ( $e:expr ) => {
         if !$e {
-            return Err(Box::new(std::io::Error::new(
-                std::io::ErrorKind::Other,
-                format!(
+            return ::std::result::Result::Err(::std::boxed::Box::new(::std::io::Error::new(
+                ::std::io::ErrorKind::Other,
+                ::std::format!(
                     "{}:{}: expectation failed: {:?} is false",
                     file!(),
```

Comment:
Also these?

Reply:
`file`, `line` and `stringify` don't seem to be available in `std` ("could not find `file` in `std`") &ndash; maybe they're special because they're built-ins?